src/www/ui/admin-upload-edit.php

```diff
@@ -105,13 +105,17 @@ public function Output()
       $text = _("Permission Denied");
       return "<h2>$text</h2>";
     }
-    $rc = $this->UpdateUploadProperties($upload_pk, $NewName, $NewDesc);
-    if ($rc == 0) {
-      $text = _("Nothing to Change");
-      $this->vars['message'] = $text;
-    } else if ($rc == 1) {
-      $text = _("Upload Properties successfully changed");
-      $this->vars['message'] = $text;
+
+    if (array_key_exists("upload_pk", $_POST)) {
+      $descChanged = array_key_exists("newdesc", $_POST);
+      $rc = $this->UpdateUploadProperties($upload_pk, $NewName, $NewDesc, $descChanged);
```

Comment:
seems like $descChanged is not required here. ity is been passed as 4th argument to a function which require only 3 arguments.